examples/plot_stochastic.py

```diff
@@ -33,7 +33,7 @@
 n_source = 7
 n_target = 4
 reg = 1
-numItermax = 1000
+num_iter_max = 1000
```

Comment:
Suggestion: `maxiter` would be more standard since it's used in scipy and sklearn ?

Reply:
Are you talking about just the plot_stochastic module ? Or every function ?

Reply:
I agree with @hichamjanati. Moreover, I think that if we change it here, we should change it everywhere.

Reply:
I think if we change it somewhere, we should change it everywhere as well.

Reply:
num_iter_max sounds really french. +1 for maxiter like sklearn

Reply:
I think maxiter is great  but Kilian this is not a one submodule change, you need to change it in the whole toolbox while mainteining compatibility with a deprecation warning.  I think it can be done with decorators where we give corresponding old_name-new name. In any case you should not change the tests (other than changing variable names), you should provide new test with the new parameters tested until we do a proper deprecation/removing of the old names.

Before rushing into things i think we should agree and discuss all variable change and how we can provide old compatibility without adding ugly ifs and having double variables in the function signature. I created a wiki page called Variable renaming where we can put all the propositions. unless there is already a document where this is discussed we really need to agree on things in order to avoid unecessary work.